snow/consensus/snowball/binary_snowflake.go

```diff
@@ -11,29 +11,37 @@ func newBinarySnowflake(alphaPreference, alphaConfidence, beta, choice int) bina
 	return binarySnowflake{
 		binarySlush:     newBinarySlush(choice),
 		alphaPreference: alphaPreference,
-		alphaConfidence: alphaConfidence,
-		beta:            beta,
+		alphaConfidence: []int{alphaConfidence},
+		beta:            []int{beta},
+		confidence:      make([]int, 1),
 	}
 }
 
 // binarySnowflake is the implementation of a binary snowflake instance
+// Invariant:
```

Comment:
Yes, I think the proper place to add this would be when verifying the parameters here: https://github.com/ava-labs/avalanchego/blob/flexible-snowflake-single-counter/snow/consensus/snowball/parameters.go#L93 as opposed to in the constructor, so that we do it once upfront.

Right now there's no need to do this because there is only a single value for each, so each statement here is vacuously true.

I'm planning to (1) implement the multicounter approach in this PR, (2) add test coverage and update constructors to take in multiple params in the next PR, and then (3) handle modifying the early termination cases as needed in the consensus engine and only then update the parameters to actually allow passing in multiple confidence/beta pairs.